objects/Wasp/Alarm_0.gml

```diff
@@ -0,0 +1,24 @@
+/// @description Fire boss bullets
+// You can write your code in this editor
+var bullet0 = instance_create_layer(x+140, y+150, "Instances", EnemyBullet);
+var bullet1 = instance_create_layer(x+140, y+150, "Instances", EnemyBullet);
+var bullet2 = instance_create_layer(x+140, y+150, "Instances", EnemyBullet);
+with (bullet0)
+{
+   direction = point_direction(x, y, Player.x, Player.y);
+   speed = 6;
+   image_angle = direction
+}
+with (bullet1)
+{
+   direction = point_direction(x, y, Player.x+180, Player.y+180);
+   speed = 6;
+   image_angle = direction
+}
+with (bullet2)
+{
+   direction = point_direction(x, y, Player.x-180, Player.y-180);
+   speed = 6;
+   image_angle = direction
+}
+alarm[0] = room_speed;
```

Comment:
room_speed is in a weird deprecation status, its fine to use it but just want to remind we have global.framerate (This also allows us to manipulate framerates in things like the pause menu without risking math mistakes) Not a big deal for this project just worth noting.

Another note (Again not likely relevant to our project ) alarms are shared and limited in number (like 10/16) so depending on how far we go they can become strained.

Again not a concern for us right now.